### DISCUSSION TITLE
No subscription popup (subscription nag) wasn't deactivated with the script

### DISCUSSION THREAD

**Discussion by logicito** [tteck/Proxmox#3740] [General]
Does anything have changed regarding disabling the "subscription nag"? I run the script but as soon as I login it was there, when I refresh for update the system, the same.

I am using Safari 18 on macOS 15, cleared the cache as well

**Comment by tteck** [comment]
Clear browser cache

**Reply by logicito** [reply]
I did, but didn't do anything

**Reply by logicito** [reply]
Now it worked, had to quit Safari 18, thank you
